blueocean-rest-impl/pom.xml

```diff
@@ -35,6 +35,10 @@
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>display-url-api</artifactId>
         </dependency>
+         <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>scm-api</artifactId>
```

Comment:
🐛  We can't add it here. scm api shouldn't be in this module. Some other class path issue (sigh)... will get to it. Lets do the right way.

Reply:
@vivek this is just to see if it changes the result of the failing test on master... which seems a brittle test

Reply:
ah ok 👍

Reply:
so - as you can see @vivek this made it pass by probably changing how the test class is loaded :(